test/unit/specs/components/v-search-results-title.spec.js

```diff
@@ -0,0 +1,19 @@
+import VSearchResultsTitle from '~/components/VSearchResultsTitle'
+import { render, screen } from '@testing-library/vue'
+
+describe('VSearchResultsTitle', () => {
+  let options = {
+    props: {
+      size: 'large',
+    },
+    scopedSlots: {
```

Comment:
Why is it `scopedSlots`, not simple `slots` here?

Reply:
I copy/pasted from another test, I wonder if it makes a difference? I can switch to `slots`.